extensions/amp-list/amp-list.md

```diff
@@ -153,11 +153,11 @@ In `<amp-list>`, you can use the [`items`](#items) attribute to render a subset
 
 ### Specifying an overflow
 
-Optionally, the `<amp-list>` component can contain an element with the `overflow` attribute. Add an element with the AMP `overflow` attribute to `amp-list` if the AMP framework is unable to size it as requested. If you include a child element of `amp-list` with the `overflow` attribute, it will appear if one of the following conditions are met:
+Optionally, the `<amp-list>` component can contain an element with the `overflow` attribute. If you include a child element of `amp-list` with the `overflow` attribute, it will appear if all of the following conditions are met:
 
--   The bottom of `amp-list` is below the viewport.
-
--   The bottom of `amp-list` is within 15% of the height of the entire page and within 1000px of the end of the page.
+-   The size specified for the `amp-list` was not large enough to fit its rendered contents.
```

Comment:
not to be too pedantic - amp-list `is` not large enough to be consistent with the remaining bullet points.

Reply:
would this first bullet point be clearer if we reversed it? As in, "The contents rendered into the `amp-list` are larger than the container?" Or, "The contents of the `amp-list` won't fit into its container?"

Actually, is it a matter of height only? In that case, "The height of the rendered contents is too large for the container"?

Reply:
@dmanek , can you say more? Are you saying that an individual `amp-list` container won't be so large that the last two bullet points get put into practice?

Reply:
I think he's just commenting on the word choice of `was` vs. `is`. I don't have the grammar knowledge to say which way is correct (I thought _was_ was fine seeing as the specified size happened _before_ render)

Reply:
reversed!